setup.py

```diff
@@ -1,42 +1,41 @@
-# Copyright (c) 2020 Aiven, Helsinki, Finland. https://aiven.io/
-
-from importlib.machinery import SourceFileLoader
-from setuptools import find_packages, setup
-
-import sys
-
-
-def get_version():
-    return SourceFileLoader("version", "aiven_db_migrate/migrate/version.py").load_module().__version__
-
+from distutils.core import setup
 
 setup(
-    author="Aiven",
-    author_email="support@aiven.io",
-    entry_points={
-        "console_scripts": [
-            "pg_migrate = aiven_db_migrate.migrate.pgmigrate:main",
-        ],
-    },
+    name='aiven_db_migrate',
+    packages=['aiven_db_migrate'],
+    version='1.0.0',
+    license='apache-2.0',
+    description='Aiven database migration tool. This tool is meant for easy migration of databases from some database '
+                'service provider, such AWS RDS, or on premises data center, to Aiven Database as a Service. However, '
+                'it\'s not limited for Aiven services and it might be useful as a generic database migration tool.',
+    author='Open Source @ Aiven',
+    url='https://github.com/aiven/aiven-db-migrate',
+
+    download_url='TBD',
+    keywords=['aiven', 'db-migrate'],
     install_requires=[
-        "psycopg2"
+        'flake8==3.8.4',
+        'isort==5.6.4',
+        'mypy==0.790',
+        'psycopg2==2.8.6',
+        'pylint==2.6.0',
+        'pytest==6.1.2',
+        'yapf==0.30.0',
     ],
-    license="Apache 2.0",
-    name="aiven-db-migrate",
-    packages=find_packages(exclude=["test"]),
-    platforms=["POSIX", "MacOS", "Windows"],
-    description="Aiven database migration tool",
-    long_description=open("README.md").read(),
-    url="https://aiven.io/",
-    version=get_version(),
     classifiers=[
-        "Development Status :: 2 - Pre-Alpha",
-        "Intended Audience :: Developers",
-        "Topic :: Software Development :: Libraries",
-        "License :: OSI Approved :: Apache Software License",
-        "Programming Language :: Python :: 3.5",
-        "Programming Language :: Python :: 3.6",
-        "Programming Language :: Python :: 3.7",
-        "Programming Language :: Python :: 3.8",
+        'Development Status :: 5 - Production/Stable',
+        'Intended Audience :: Developers',
+        'License :: OSI Approved :: Apache license 2.0',
+        'Programming Language :: Python :: 3.0',
+        'Programming Language :: Python :: 3.1',
+        'Programming Language :: Python :: 3.2',
+        'Programming Language :: Python :: 3.3',
+        'Programming Language :: Python :: 3.4',
```

Comment:
Readme mentions 3.5+. Classifiers should match.

Might also be worth considering dropping support for out-of-life Python versions.